### PR TITLE
New Backgrounds and Main Menu Text Fix

### PR DESCRIPTION
**Pull request by Kefpull**
moved background loading and calculation to a method (still part is in static, which may be dangerous)

## Release Notes

- **New Features**
	- Introduced a new high-resolution main menu background setting for enhanced customization.
	- Added functionality to load background images based on resolution preferences.

- **Improvements**
	- Updated main menu logo color for better visibility.
	- Enhanced splash screen logo texture during loading.
	- Streamlined logic for theme management and rendering in the main menu.

These updates aim to improve user experience and visual appeal within the application.

### CODE REVIEW

**Comment by Kefpull** [comment]
@coderabbitai full review

File: src/main/java/keystrokesmod/mixins/impl/gui/MixinGuiMainMenu.java [hunk 1]
```diff
@@ -35,14 +40,24 @@
...
     protected abstract boolean func_183501_a();
 
+    static{
+        if(clientTheme.highResMainMenuBackgrounds.isToggled()) {
+            BackgroundUtils.loadBackgroundImages(true);
+        }
+        else{
+            BackgroundUtils.loadBackgroundImages(false);
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Kefpull) [reply src/main/java/keystrokesmod/mixins/impl/gui/MixinGuiMainMenu.java]: yes, good job code rabbit ai

File: src/main/java/keystrokesmod/module/impl/render/ClientTheme.java [hunk 2]
```diff
@@ -40,6 +41,8 @@ public ClientTheme() {
         this.registerSetting(buttonLowerCase = new ButtonSetting("Button lower case", false, button::isToggled));
         this.registerSetting(background = new ButtonSetting("Background", true));
         this.registerSetting(mainMenu = new ButtonSetting("Main menu", true));
+        this.registerSetting(highResMainMenuBackgrounds = new ButtonSetting("Only High Res Backgrounds", false, mainMenu::isToggled));
+
```
[the remark this answers is not in the document]
Comment (Kefpull) [reply src/main/java/keystrokesmod/module/impl/render/ClientTheme.java:45]: the new ones are an okay resolution, 720p, but more importantly they are highly compressed, (as low as 39kb vs the 1681 kb for 5.png)

**Comment by xia-mc** [comment]
Some backgrounds are not enough perfect..

**Comment by Kefpull** [comment]
@xia-mc I agree, if you possibly have the thumbnails still from those videos then it would be good to replace them and move thew updated pictures to the "high quality" section. Otherwise just setting highResolutionOnly as true in default would also work

**Pull request closed without merging**